crates/bevy_sprite/src/dynamic_texture_atlas_builder.rs

```diff
@@ -4,19 +4,30 @@ use bevy_math::{IVec2, Rect, Vec2};
 use bevy_render::texture::{Image, TextureFormatPixelInfo};
 use guillotiere::{size2, Allocation, AtlasAllocator};
 
+/// Helper utility to update [`TextureAtlas`] on the fly.
+/// It is used in cases when texture is created highly dynamic,
+/// e.g: font glyph texture, only render for letters to be rendered.
 pub struct DynamicTextureAtlasBuilder {
-    pub atlas_allocator: AtlasAllocator,
-    pub padding: i32,
+    atlas_allocator: AtlasAllocator,
+    padding: i32,
 }
 
 impl DynamicTextureAtlasBuilder {
+    /// Create a new [`DynamicTextureAtlasBuilder`]
+    ///
+    /// # Arguments
+    ///
+    /// * `size` - total size for the atlas
+    /// * `padding` - padding between textures in the atlas
```

Comment:
Is this on all sides?

Reply:
sorry, I do not understand this.. what do you mean exactly?

Reply:
There are two reasonable alternatives for this parameter:
- the padding is the gap between images
- the padding is the empty pixels added around each image

These result in different width "gutters", so it's helpful to know which behavior is used.

Reply:
Ah, I see. I should not explain padding with padding.